test/template-test.js

```diff
@@ -0,0 +1,37 @@
+import template from '../src/template'
+
+
+describe('template', () => {
+  it('execute', (done) => {
+    let format = '${timestamp} (${namespace}) ${level}: ${data}';
```

Comment:
for this variable and other below, if the variable doesn't change, please use `const`